programs/marginfi/src/state/marginfi_group.rs

```diff
@@ -1041,23 +1088,27 @@ assert_struct_align!(BankConfig, 8);
 #[derive(Debug)]
 /// TODO: Convert weights to (u64, u64) to avoid precision loss (maybe?)
 pub struct BankConfig {
-    pub asset_weight_init: WrappedI80F48,
-    pub asset_weight_maint: WrappedI80F48,
+    pub asset_weight_init: WrappedI80F48,  // 288 - 304
+    pub asset_weight_maint: WrappedI80F48, // 304 - 320
 
-    pub liability_weight_init: WrappedI80F48,
-    pub liability_weight_maint: WrappedI80F48,
+    pub liability_weight_init: WrappedI80F48,  // 320 - 336
+    pub liability_weight_maint: WrappedI80F48, // 336 - 352
 
-    pub deposit_limit: u64,
+    pub deposit_limit: u64, // 352 - 360
 
-    pub interest_rate_config: InterestRateConfig,
-    pub operational_state: BankOperationalState,
+    pub interest_rate_config: InterestRateConfig, // 360 - 600
+    pub operational_state: BankOperationalState,  // 600
 
-    pub oracle_setup: OracleSetup,
-    pub oracle_keys: [Pubkey; MAX_ORACLE_KEYS],
+    pub oracle_setup: OracleSetup,              // 601
+    pub oracle_keys: [Pubkey; MAX_ORACLE_KEYS], // 602 - 762
+    // Note: Key 0 at 602 - 634, and so forth.
 
-    pub borrow_limit: u64,
+    // Note: Pubkey is aligned 1, so borrow_limit is the first aligned-8 value after deposit_limit
+    pub _pad0: [u8; 6], // Bank state (1) + Oracle Setup (1) + 6 = 8
 
-    pub risk_tier: RiskTier,
+    pub borrow_limit: u64, // 768 - 776
+
+    pub risk_tier: RiskTierU64, // 776 - 784
```

Comment:
Would it be breaking to expand this field into a u8 enum + 3 padding bytes, instead of introducing this custom struct?

Reply:
Yeah my thinking is that this way the change would be transparent wherever this field is consumed